docs/learning/howto/use-roi-metrics.md

```diff
@@ -0,0 +1,202 @@
+# Using ROI Metric Data
+
+**ROI Metrics Data** stores arbitrary key value pairs against your job to help understand your ROI per job execution. The metrics will be stored against the each execution of the job, and accessible via API for further analysis.  This article will walk through a setup scenario and provide some helpful example scripts to gather data from the API.
+
+## Configuring the Metrics
+
+### Example Scenario:
+
+Customer X has decided to track how often time is saved with their newly implemented Automation job (uniquely titled `ROI Job`).  They know from historical issues that doing the same work prior to writing this job the manual steps took around 75 minutes across all teams, start to finish.  The job now runs in just seconds.  To configure the job follow the steps or download the job definition.
+
+:::: tabs
+::: tab Steps to Create
+
+
+1. Create a new Job.
+1. Title job `ROI Job`.
+1. On the Workflow Tab add a single `Command` step with the command `sleep 5`.  (Assumes this will be running on Linux host)
+1. Choose the _Execution Plugins_ tab.
+1. Check the box for **ROI Metrics Data**.<br><br>
+    ![ROI Metrics Plugin](~@assets/img/roi-metrics-data.png)
+1. Click **Add Custom Field**.<br><br>
+    ~[Add Custom Field](~@assets/img/roi-metrics-customfield.png)
+1. Enter a value for:
+    - ***Field Label***: `Hours Saved`
+    - ***Field Key***: `hours`
+    - ***Description***: `FTE hours saved each time this job is run.`
+1. Click **Add**.
+1. Enter a value of `1.25` for the metric in the **Hours Saved** field.
+1. Save the job.
+1. Run the job and look for the ROI Metrics Data tab.  Confirm that the data is populated.
+1. Run the job a few more times so we have multiple executions.
+
+:::
+::: tab Download Job Definition
```

Comment:
could we save this job definition in git, and provide link to download the .yaml file? This is much simpler than copy/paste and making sure they save as .yaml

Reply:
We could do that in a future iteration.  I don't want to keep creating one-off repos for that stuff.  Let's figure out a content strategy for it.